src/ucp/core/ucp_ep.h

```diff
@@ -135,6 +135,8 @@ typedef struct ucp_ep_config {
 
     /* Error handling mode */
     ucp_err_handling_mode_t    err_mode;
+    /* Status of the endpoint */
+    ucs_status_t               status;
```

Comment:
this should not be here - see comment about err_mode. what if u have 2 endpoints with same config, on of them fails and other one is still OK?